distributed/protocol/serialize.py

```diff
@@ -195,6 +202,14 @@ def deserialize(self):
         frames = decompress(self.header, self.frames)
         return deserialize(self.header, frames)
 
+    def __eq__(self, other):
+        return (isinstance(other, Serialized) and
+                other.header == self.header and
+                other.frames == self.frames)
+
+    def __ne__(self, other):
+        return not (self == other)
```

Comment:
Ok, so I think I'll remove `Serialized.__eq__` (which doesn't seem to be needed actually) and also implement `Serialize.__hash__`.